src/test/kotlin/org/rust/ide/annotator/RsErrorAnnotatorTest.kt

```diff
@@ -4456,4 +4458,30 @@ class RsErrorAnnotatorTest : RsAnnotatorTestBase(RsErrorAnnotator::class) {
             }
         }
     """)
+
+    fun `test do not annotate usage of private field in debugger code fragment`() = checkByCodeFragment("""
+        mod my {
+            pub struct Foo { inner: i32 }
+        }
+        fn bar(foo: my::Foo) {
+            /*caret*/;
+        }
+    """, """foo.inner""", ::RsDebuggerExpressionCodeFragment)
+
+    @ProjectDescriptor(WithStdlibRustProjectDescriptor::class)
+    fun `test do not annotate usage of Vec private field in debugger code fragment`() = checkByCodeFragment("""
+        fn main() {
+            let xs = vec![1,2,3];
+            /*caret*/;
+        }
+    """, """xs.inner""", ::RsDebuggerExpressionCodeFragment)
```

Comment:
Are you sure that we actually need this test? Looks like `test do not annotate usage of private field in debugger code fragment` covers necessary case